### PR TITLE
STM32F1: Only include USBComposite if flagged

### PR DESCRIPTION
**Pull request by tpruvot**

### CODE REVIEW

File: Marlin/src/HAL/HAL_STM32F1/msc_sd.h [hunk 1]
```diff
@@ -16,9 +16,13 @@
...
+#ifdef USE_USB_COMPOSITE
+
 #include <USBComposite.h>
 
 extern USBMassStorage MarlinMSC;
 extern USBCompositeSerial MarlinCompositeSerial;
 
 void MSC_SD_init();
+
+#endif
```
Comment (thinkyhead) [inline Marlin/src/HAL/HAL_STM32F1/msc_sd.h]: It's better not to wrap headers. Instead, wrap the `#include` of the header.

Comment (tpruvot) [reply Marlin/src/HAL/HAL_STM32F1/msc_sd.h]: maybe to move after #ifdef SERIAL_USB ? the include msc_sd.h ?

Comment (tpruvot) [reply Marlin/src/HAL/HAL_STM32F1/msc_sd.h]: hmm unsure, because serial and sdcard share could be 2 different things

Comment (tpruvot) [reply Marlin/src/HAL/HAL_STM32F1/msc_sd.h]: > It's better not to wrap headers. Instead, wrap the `#include` of the header.

ok, done

**Pull request merged**